pkg/controller/codes/bulk_issue.go

```diff
@@ -39,7 +46,8 @@ func (c *Controller) HandleBulkIssue() http.Handler {
 		currentRealm := membership.Realm
 
 		if !currentRealm.AllowBulkUpload {
-			controller.Unauthorized(w, r, c.h)
+			flash.Error("That feature is not enabled for your realm!")
```

Comment:
Nit

```suggestion
			flash.Error("Bulk upload is not enabled for your realm!")
```

Reply:
I'm intentionally doing this so we don't leak the feature.